### DISCUSSION TITLE
Is there exist a plugin to support url like http://example.com?category_id=2&category_id=3 ?

### DISCUSSION THREAD

**Discussion by zw963** [jeremyevans/roda#219] [General]
Let me ask the first question, haha

http://example.com?category_id=2&category_id=3

When use with roda, will get params like `params={category_id="3"}`, but what i expected is 
`params={category_id=["3", "4"]}`, so, if we support this feature? (like rails)

Thank you.

**Comment by jeremyevans** [comment]
There isn't currently a plugin for that.  That's an anti pattern, IMO.  If you want an array parameter, add `[]` after the parameter name: `http://example.com?category_id[]=2&category_id[]=3`.  This has the advantage that in the single parameter case, you still get an array.


While I don't have plans on working on such a plugin, it's possible a plugin could be added to support it, depending on whether the community is in favor of it.

**Reply by zw963** [reply]
It works!  i guess it me misunderstood somethings, category_id[] is more elegance, thank you!
